### DISCUSSION TITLE
Commits to GitHub are not verified.

### DISCUSSION THREAD

**Discussion by kushanksriraj** [codesandbox/codesandbox-client#5619] [Q&A]
Hello everyone!

I recently started using codesandbox for personal react projects and I'm very happy with it. 

Just one issue I'm facing is that the commits I do from codesandbox to GitHub are not shown as "Verified" and therefore these commits are not showing on the heatmap on GitHub profile.

Is there a fix for this or it's just as it is?

**Comment by nilskroe** [comment]
Any update here?

**Comment by JamesACS** [comment]
Hello @nilskroe 

An option to enforce git commit signing is on our roadmap, you can keep up to date with the latest on this on our [community](https://www.codesandbox.community/c/feature-requests/signed-git-commits)
